grails-app/controllers/au/org/ala/biocollect/merit/ActivityController.groovy

```diff
@@ -55,6 +58,14 @@ class ActivityController {
     private Map activityAndOutputModel(activity, projectId) {
         def model = activityModel(activity, projectId)
         addOutputModel(model)
+        addProjectActivity(model,activity)
+        model
+    }
+
+    def addProjectActivity(model, activity){
```

Comment:
No need for a return type here: the 'model' map is passed by reference, so the method will update it. You could change the return type to void and remove the last line.